### ISSUE TITLE
Alone in the Dark: A New Nightmare (PAL)

### ISSUE THREAD

**Issue by hatoving** [Rinnegatamante/flycast-vita-compatibility-list#65]
Game runs decently (20-28 FPS depending on the scene). Although what's weird is that it ran full-speed on a previous build of Flycast, and it doesn't have that annoying audio click. That audio click happens everywhere, especially at cutscenes. Otherwise, there aren't any graphical issues.

![Screenshot 2022-08-09 20-27-25](https://user-images.githubusercontent.com/64536760/183986533-b4fd3780-e2cc-4db7-a14d-ca81eb9cb261.png)

Settings used:
Transparent Sorting -> Per Strip
Auto Frame Skipping -> Maximum
Simple Shaders, Mipmaps and Neon SIMD -> On
Self-Mod. Code Checks -> Reduced
Downclock Ratio -> 1.542

Label: In-game+

**Comment by hatoving** [comment]
With the new Float Ops option on, it's now mostly fullspeed (with cutscenes still being very slow).
The audio clicking seems to be gone but I don't know at this point lmao

**Comment by Rinnegatamante** [comment]
Ingame+ still?

**Comment by hatoving** [comment]
Yeah, I guess so. It still stutters just a little bit, but again, it stutters at the cutscenes (which happen pretty commonly).
